tony-history-server/README.md

```diff
@@ -43,4 +43,13 @@ Before using the script, ensure that you have set execution permission (`chmod +
 $ ./buildAndDeploy.sh user hostname.test.abc.com
 ```
 
-See [script](./buildAndDeploy.sh) for more.
\ No newline at end of file
+See [script](./buildAndDeploy.sh) for more details.
+
+- To start the server after bundling with `gradle dist`:
+1. `unzip $root/build/distributions/playBinary.zip`
```

Comment:
Can you rename the zip `tony-history-server`?

Reply:
Yep.